src/orion/plotting/backend_plotly.py

```diff
@@ -549,6 +549,318 @@ def build_frame():
+                exp_parallels["names"].append(name)
+                exp_parallels["n_workers"].append(experiment.executor.n_workers)
+                exp_parallels["objective"].append(dfs["objective"].tolist()[0])
+
+        return pd.DataFrame(exp_parallels)
+
+    def get_objective_name(experiments):
+        """Infer name of objective based on trials of one experiment"""
+        if compute_average and isinstance(experiments, dict):
+            return get_objective_name(sum(map(list, experiments.values()), []))
+
+        if isinstance(experiments, dict):
+            experiments = experiments.values()
+
+        for experiment in experiments:
+            trials = experiment.fetch_trials_by_status("completed")
+            if trials:
+                return trials[0].objective.name
+        return "objective"
+
+    if not experiments:
+        raise ValueError("Parameter 'experiment' is None")
+
+    df = build_group()
+    fig = go.Figure()
+
+    if df.empty:
+        return fig
+
+    names = set(df["names"])
+    for i, name in enumerate(sorted(names)):
+        parallel_data = df[df["names"] == name]
+        if "objective_mean" in parallel_data:
+            y = parallel_data["objective_mean"]
+            durations = parallel_data["duration_mean"]
+        else:
+            y = parallel_data["objective"]
+            durations = parallel_data["duration"]
+
+        x = parallel_data["n_workers"]
+
+        fig.add_scatter(
+            x=x,
+            y=y,
+            mode="lines+markers",
+            name=name,
+            customdata=list(zip(durations, parallel_data["n_workers"])),
+            hovertemplate=_template_workers(name),
+        )
+
+    fig.update_layout(
+        title=f"Parallel Advantage",
+        xaxis_title=f"Number of workers",
+        yaxis_title=get_objective_name(experiments),
+        hovermode="x",
+    )
+
+    return fig
+
+
+def durations(experiments, with_evc_tree=True, order_by="completed", **kwargs):
+    """Plotly implementation of `orion.plotting.regrets`"""
+
+    compute_average = bool(
+        isinstance(experiments, dict)
+        and isinstance(next(iter(experiments.values())), Iterable)
+    )
+
+    def build_groups():
+        """Build dataframes for groups of experiments"""
+        if compute_average:
+            data_frames = dict()
+            for name, group in experiments.items():
+                df = orion.analysis.average(
+                    build_frame(group), keys=["best", "duration"]
+                )
+                df["name"] = [name] * len(df)
+                data_frames[name] = df
+            data_frames = pd.concat(data_frames)
+        elif isinstance(experiments, dict):
+            data_frames = build_frame(experiments.values(), list(experiments.keys()))
+        else:
+            data_frames = build_frame(experiments)
+
+        return data_frames
+
+    def build_frame(experiments, names=None):
```

Comment:
Maybe this function could also be replaced by regrets() with a custom function for the x axis. This could be possible if we rely on the leaner version without buttons as discussed [here](https://github.com/Epistimio/orion/pull/685/files#r774163445).

Reply:
Also done in 060b74782dd90141c152f2aa434964c241e46b38. Thanks! :)